public/js/report.js

```diff
@@ -975,40 +975,45 @@ var SideBar = React.createClass({
     },
 
     /**
+     * Dynamically create form and submit.
+     */
+    postForm: function ($a, sequence_ids) {
+      var form = $("<form action=\"/get_sequence\" method=\"post\"></form>")
```

Comment:
+1 for dynamic form creation - one function abstracts away the entire process. I had a static form in mind (generated through Report's render method) whose field would be updated, submitted, and cleared on click.

Consider breaking the string using other jquery methods. I don't know if all jquery methods are available to an unattached dom node created this way. But if `prop` is, that should do: `$('<form/>').prop('action', ...).prop('method', ...)`.